mino/minogrpc/controller/mod_test.go

```diff
@@ -20,11 +26,20 @@ func TestMinimal_Build(t *testing.T) {
 }
 
 func TestMinimal_OnStart(t *testing.T) {
-	minimal := NewMinimal()
+	dir, err := ioutil.TempDir(os.TempDir(), "minogrpc")
```

Comment:
I know you're going to hate me at this point, but I'd still favor small tests with descriptive names :)